docs/database/server/server.md

```
@@ -436,15 +436,15 @@ channel_ids string
 user_access_level number
 user_ids string
 team_ids string
+is_default bool
 
 
 AiThread
```

markdownlint-cli2 (0.23.2)

[warning] Heading style
Expected: atx; Actual: setext

(MD003, heading-style)